react/components/Router.js

```diff
@@ -9,7 +9,7 @@ import ExtensionPoint from '../ExtensionPoint'
 import IntrospectionFetcher from './IntrospectionFetcher'
 
 function isRelative(path) {
-  return path[0] !== '/'
+  return !path.startsWith('http://') && !path.startsWith('https://')
```

Comment:
Why was this change necessary?

Reply:
Because `isRelative('https://unpkg.com/react-intl@2.4.0/locale-data/en.js')` being true doesn't make sense.

But I was looking where this function was being used and I think this is legacy code:

```js
function prefix(path) {
  const prefix = isRelative(path) ? '/assets/' : ''
  return `${prefix}${path}`
}
```

We don't even have this `/assets/*` route in render-server anymore.

I'll remove this code.